shared_model/cryptography/keypair.hpp

```diff
@@ -43,49 +45,28 @@ namespace shared_model {
       using PrivateKeyType = PrivateKey;
 
       explicit Keypair(const PublicKeyType &public_key,
```

Comment:
Should it be explicit?

Reply:
it was explicit before. I do not change interface or anything, just separate definition and declaration.